src/pages/LoadingPage.tsx

```diff
@@ -65,7 +69,36 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ onLoad, logType }) => {
       dispatchToast.error(error);
     }
   }, [data, ingestLines, error, onLoad, logType, dispatchToast]);
-  return <div>I am the loading page</div>;
+  return (
+    <StyledPageLayout>
+      {isLoading || !error ? (
+        <LoadingBarContainer>
+          <LogoContainer>🌿 Loading Lobster...</LogoContainer>
+          <LoadingBar indeterminate progress={100} />
+        </LoadingBarContainer>
+      ) : (
+        <div>404 here</div>
+      )}
+    </StyledPageLayout>
+  );
 };
 
+const LoadingBarContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 40%;
+`;
+
+const LogoContainer = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: ${size.xs};
+`;
+const StyledPageLayout = styled(PageLayout)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
```

Comment:
Can you make this a `div` rather than a styled `PageLayout`? I think it's a little strange to have a `PageLayout` nested in a `PageLayout`

(You may have to add `height: 100%`)